chunked_upload/settings.py

```diff
@@ -35,7 +35,14 @@ def default_upload_to(instance, filename):
 
 
 # Storage system
-STORAGE = getattr(settings, 'CHUNKED_UPLOAD_STORAGE_CLASS', lambda: None)()
+USE_TEMP_STORAGE = getattr(settings, 'CHUNKED_UPLOAD_USE_TEMP_STORAGE', False)
+# Use django default or via settings defined storage
+if not USE_TEMP_STORAGE:
+    STORAGE = getattr(settings, 'CHUNKED_UPLOAD_STORAGE_CLASS', lambda: None)()
+# Use temporary storage for chunks
+else:
+    from chunked_upload.storages import TemporaryFileStorage
+    STORAGE = TemporaryFileStorage()
```

Comment:
@GitRon 
If I set `CHUNKED_UPLOAD_USE_TEMP_STORAGE` to `True`, the chunks will be uploaded to the temporary location and it won't be uploaded to AWS S3 either during the upload or upload completion. Did we miss something?

Reply:
@jerinpetergeorge S3 is an object not a file storage. So you cannot write to it like a local hard drive. What I changed is that the chunks are written to the local hard drive and when the stuff is done, it gets moved to S3.